test/jdk/java/lang/StackWalker/CallerSensitiveMethod/csm/jdk/test/CallerSensitiveTest.java

```diff
@@ -42,29 +37,12 @@
  * StackWalker::getCallerClass can't be called from @CallerSensitive method.
  */
 public class CallerSensitiveTest {
-    static final Policy DEFAULT_POLICY = Policy.getPolicy();
     private static final String NON_CSM_CALLER_METHOD = "getCallerClass";
     private static final String REFLECTIVE_GET_CALLER_METHOD = "getCallerClassReflectively";
     private static final String CSM_CALLER_METHOD = "caller";
 
     public static void main(String... args) throws Throwable {
-        boolean sm = false;
-        if (args.length > 0 && args[0].equals("sm")) {
-            sm = true;
-            PermissionCollection perms = new Permissions();
-            perms.add(new RuntimePermission("getStackWalkerWithClassReference"));
-            Policy.setPolicy(new Policy() {
-                @Override
-                public boolean implies(ProtectionDomain domain, Permission p) {
-                    return perms.implies(p) ||
-                        DEFAULT_POLICY.implies(domain, p);
-                }
-            });
-            System.setSecurityManager(new SecurityManager());
-        }
-
-        System.err.format("Test %s security manager.%n",
-                          sm ? "with" : "without");
+        System.err.println("Test without security manager.");
```

Comment:
Security manager is not relevant any more.  Suggest to drop this println.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/002276450e625b66b786fb7eae7256bbcafa7496